extension/src/openvic-extension/singletons/MenuSingleton.cpp

```diff
@@ -486,7 +487,44 @@ Dictionary MenuSingleton::get_province_info_from_index(int32_t index) const {
 	if (max_employee_count == 0) {
 		ret[province_info_rgo_employment_percentage_key] = 100.0f;
 	} else {
-		ret[province_info_rgo_employment_percentage_key] = (rgo.get_total_employees_count_cache() * fixed_point_t::_100() / max_employee_count).to_float_rounded();
+		ret[province_info_rgo_employment_percentage_key] =
+			(rgo.get_total_employees_count_cache() * fixed_point_t::_100() / max_employee_count).to_float_rounded();
+	}
+
+	if (rgo.is_valid()) {
+		String amount_of_employees_by_pop_type;
+		for (auto const& [pop_type, employees_of_type] : rgo.get_employee_count_per_type_cache()) {
+			if (employees_of_type > 0) {
+				amount_of_employees_by_pop_type +=
+					"  -" + GUILabel::get_colour_marker() + "Y" +
+					tr(Utilities::std_to_godot_string(pop_type.get_identifier())) + GUILabel::get_colour_marker() + "!:" +
+					String::num_int64(employees_of_type) + "\n";
```

Comment:
```suggestion
					String::num_int32(employees_of_type) + "\n";
```

Reply:
There is no such function, `godot::String` only has `num(double)`, `num_int64(int64_t)` and `num_uint64(uint64_t)`